variables.tf

```diff
@@ -91,9 +91,14 @@ variable "public_subnet_tags" {
   description = "Additional tags to set on the public subnets"
 }
 
+variable "private_dynamodb_endpoint" {
+  default     = false
+  description = "Deploy a DynamoDB endpoint for your private subnets"
+}
+
 variable "private_s3_endpoint" {
   default     = false
-  description = "Deploy an S3 endpoint for your private subnets"
+  description = "Deploy a S3 endpoint for your private subnets"
```

Comment:
Can you revert this please change please, "an" is correct for "S3."

Reply:
Done. What's the rule for that actually? I thought you use "an" only for words starting with a vowel.